psl/psl-core/src/builtin_connectors/sqlite_datamodel_connector.rs

```diff
@@ -66,8 +66,8 @@ impl Connector for SqliteDatamodelConnector {
         unreachable!("No native types on Sqlite");
     }
 
-    fn default_native_type_for_scalar_type(&self, _scalar_type: &ScalarType) -> NativeTypeInstance {
-        NativeTypeInstance::new(())
+    fn default_native_type_for_scalar_type(&self, _scalar_type: &ScalarType) -> Option<NativeTypeInstance> {
+        None
     }
```

Comment:
Similarly, now that this method is called for all connectors when constructing quaint columns, the function's signature was changed to better signal when a scalar type has no default native type.